src/server/server.js

```diff
@@ -82,12 +82,14 @@ app.use(function* (next) {
         return;
     }
 
+    /*
```

Comment:
why didn't you just remove the code outright?

Reply:
I thought I'd be doing more before pulling the wip tag.